types/index.d.ts

```diff
@@ -151,6 +153,17 @@ declare namespace StyleDictionary {
   interface Filters {
     [name: string]: Matcher;
   }
+
+  interface ParserOptions {
+    contents: string;
+    filePath: string;
+  }
+
+  interface Parser {
+    pattern: RegExp;
+    parse: (props: ParserOptions) => string | undefined;
```

Comment:
I'm not sure whether a Parser always must return a string in all circumstances, hence `string | undefined`.

Reply:
Parser should always return a plain JS object

Reply:
I'm a little amateur at TypeScript, but I believe it should be...

`parse: (props: ParserOptions) => Properties;`

Reply:
Thanks @chazzmoney, I've made this change. That makes a lot more sense!